src/ex3/utils.js

```diff
@@ -0,0 +1,72 @@
+import {ItemManager} from "./ItemManager.js";
+import {PokemonClient} from "./PokemonClient.js";
+import chalk from "chalk";
+import fs from "fs/promises";
+import fetch from "node-fetch";
+import dotenv from "dotenv";
+
+const itemManager = new ItemManager();
+const pokemon = new PokemonClient();
+const FILE='./listTodo.txt';
+const CHOICE = {
+  ID: "i",
+  NAME: "n"
+};
+const CHOICE_ARG_DESCRIPTION =
+  "Choose how to delete - either i for id or n for name";
+
+dotenv.config();
+globalThis.fetch = fetch;
```

Comment:
it's better to import and export relevant variables instead of using the globalThis functionality